src/openrct2-ui/windows/Land.cpp

```diff
@@ -356,6 +356,8 @@ static void window_land_paint(rct_window* w, rct_drawpixelinfo* dpi)
         y = w->y + previewWidget->top;
         int32_t sprite = gLandToolSize % 2 == 0 ? SPR_G2_MOUNTAIN_TOOL_EVEN : SPR_G2_MOUNTAIN_TOOL_ODD;
         gfx_draw_sprite(dpi, sprite, x, y, 0);
+        widget_draw(dpi, w, WIDX_DECREMENT);
```

Comment:
I thought you wanted to tweak the sprites, rather than draw twice?

Reply:
That was my original intent, yes, but I realised that it might be problematic if we ever want to adjust the container width. Suddenly, there would be pixels missing. This is a solution without a noticeable performance impact, anyway.